UM/Application.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, name: str, version: str, build_type: str = "", is_debug_mode:
         self._is_headless = False #type: bool
         self._use_external_backend = False #type: bool
 
-        self.CONFIG_LOCK_FILENAME = "{name}.lock".format(name = self._app_name)
+        self.CONFIG_LOCK_FILENAME = "{name}.lock".format(name = self._app_name) # type: str
```

Comment:
Out of interest, why not ```self._config_lock_filename```? IE: why is this not a private variable like any other?

Reply:
Oh, yes. I just copied as it was in the ContainerRegistry. It makes sense to change it.